web/bfex/components/data_pipeline/tasks/keywords.py

```diff
@@ -0,0 +1,73 @@
+from bfex.components.data_pipeline.tasks import Task
+from bfex.components.scraper.scrapp import Scrapp
+from bfex.models import *
+from bfex.common.exceptions import WorkflowException
+from bfex.components.key_generation.rake_approach import *
+
+
+class UpdateKeywordsFromScrape(Task):
+    """
+    Updates Keywords of a Faculty Members data in elastic.
+    """
+    def __init__(self):
+        self.task_name = "Update Keywords From Scrape"
+
+    def is_requirement_satisfied(self, data):
+        satisfied = True
+
+        if (not isinstance(data, tuple) or
+                not isinstance(data[0], str) or
+                not isinstance(data[1], Scrapp)):
+            satisfied = False
+
+        return satisfied
+
+
+    def run(self,data):
+       # professor_id = data[0]
+        #scrapp = data[1]
+        for faculty in data:
+            print(faculty.faculty_id)
```

Comment:
we should be printing something every time this runs but should we be printing this all?